### ISSUE TITLE
CRT-Royale doesn't seem to work

### ISSUE THREAD

**Issue by romraptor** [DerKoun/bsnes-hd#119]
I'm using BSNES HD 114.3 on macOS Ventura.

**Comment by DerKoun** [comment]
The latest version is bsnes-hd beta 10.6, so you might be confusing that with bsnes.
Could you describe the issue in more detail? What does the result look like? Do any other shaders work? Does bsnes-hd behave differently than bsnes?

**Comment by romraptor** [comment]
Yes, I have BSNES HD beta 10.6. I just clicked the wrong About menu.
Yes, all other shaders work. If I choose CRT-Royale the game keeps running but screen turns black.
If I use ordinary BSNES via OpenEmu all shaders work.

**Comment by DerKoun** [comment]
Does it depend on the scale factor or widescreen setting? Some shaders struggle with that in various ways.

**Comment by romraptor** [comment]
If a load a regular non-widescreen game I get the same result with CRT-Royale in BSNES HD.

Which settings more specifically should I try to disable? You talking about all the ones in "Enhancements"?

**Comment by DerKoun** [comment]
Setting Scale to disabled or 1x and also Widescreen to none should be enough.

**Comment by romraptor** [comment]
Changing those two settings didn't make a difference.

**Comment by DerKoun** [comment]
OK, so it doesn't work with bsnes-hd even at 'normal' resolution. Thanks for trying.
I'm afraid unless someone who knows about shaders helps out I won't be able to fix this.
